src/styles/blocks/page.scss

```diff
@@ -0,0 +1,13 @@
+.page {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: $primary-text-color;
+
+  transition: all 300ms ease-in-out;
```

Comment:
Avoid using the `all` keyword in transitions. It's not performant as it will apply the transition to all properties, including those that do not need to change. Specify which properties you want to transition instead.